### ISSUE TITLE
[Bug] Duplication in Rule set headings

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1725]
The new HTML and ARIA headings in the Rule Sets descriptions contain typos and should be quickly fixed. 
Introduced when 4.1.1 was removed.

- [x] removes duplication of HTML ~~HTML Messages (NA)~~
- [x] remove duplication of ARIA ~~ARIA Messages (NA)~~
- [x] add 4.1.1 description

### Existing Rule set headings
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM1pXQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cfefe58c393d3ac830dc7fd8e56295332be2568c/image.png)

**Comment by philljenkins** [comment]
Headings should now read:
- HTML specifications (NA)
- ARIA specification (NA)

Note: (NA) is not applicable for WCAG level A or AA

**Comment by philljenkins** [comment]
Code changes committed to branch dev-1676 and PR #1705
